ARCHITECTURE.md

```diff
@@ -136,7 +136,7 @@ scheduler plugin, which serves these requests on port `10299`. Each request sent
 
 In general, a `PluginResponse` primarily provides a `Permit`, which grants permission for the
 `autoscaler-agent` to assign the VM some amount of resources. By tracking total resource allocation
-on each node, the scheduler can prevent
+on each node, the scheduler can prevent a scale up request to avoid having undesired over-commit.
```

Comment:
I'd probably say the scheduler can "reject" a scale up request rather than "prevent" it. But otherwise, lgtm - thanks!